airflow/utils/state.py

```diff
@@ -36,9 +36,12 @@ class TerminalTIState(str, Enum):
     """States that a Task Instance can be in that indicate it has reached a terminal state."""
 
     SUCCESS = "success"
-    FAILED = "failed"
+    FAILED = "failed"  # This state indicates that we attempt to retry.
```

Comment:
This enum is used in lots of other places, so we can't add this comment here really.
```suggestion
    FAILED = "failed"
```

Reply:
Sure, its added in the usage so all good